rollup.config.js

```diff
@@ -12,6 +12,19 @@ import css from "rollup-plugin-css-only";
 
 const production = !process.env.ROLLUP_WATCH;
 
+function extractLastUpdatedTime(str) {
+  const tinydate = require("tinydate");
+  const json = JSON.parse(str);
+  const lastUpdatedDate = new Date(json["lastUpdate"]);
+
+  const stamp = tinydate("{MMMM} {DD} {YYYY}", {
+    MMMM: (d) => d.toLocaleString("default", { month: "long" }),
+    DD: (d) => d.getDate(),
+  });
+
+  return stamp(lastUpdatedDate);
+}
```

Comment:
A couple things here (in addition to what @jcads said about eslint, thanks!):

1. You should import tinydate via ES6 import rather than require (same as the other stuff on top)
2. I think we can just use standard javascript APIs here to extract the date, which is probably best since IIRC we're not explicitly depending on tinydate (and thus it might go away unexpectedly in the future). Try using [Date.parse](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse) to get the unix timestamp of the date and [Date.setTime](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setTime))